documentation/introduction/introduction-compatibility.asciidoc

```diff
@@ -0,0 +1,47 @@
+---
+title: Compatibility and Versioning
+order: 3
+layout: page
+---
+= Compatibility and Versioning
+
+== Compatibility
+
+Vaadin is designed to be a one-stop solution to create web UIs, but you can also use some of its parts in isolation.
+
+For the best effect, we recommend that you combine Vaadin with a well-documented Java back-end framework such as Spring, JavaEE/CDI, or OSGi.
+With these combinations, Vaadin can provide everything you need for fully featured web applications.
+
+[cols="1,2",options=header]
+|===
+|Category|Products
+|JDK|Java 8 or later; OpenJDK, Amazon Corretto, Oracle JDK
+|Programming Languages|Java, Groovy, Kotlin, etc.
+|Java Frameworks|Spring, JavaEE/CDI, OSGi, etc.
+|IDEs|Eclipse IDE, NetBeans IDE, IntelliJ IDEA, etc.
+|Servlet Containers|Apache Tomcat, Apache TomEE, Wildfly, Glassfish, JBoss, Jetty, etc.
+|Server-Side Build Management|Maven, Ivy, Gradle
+|Client-Side Build Management|Node/NPM
+|Browsers|Firefox, Chrome, Safari, IE11, Edge
```

Comment:
I say leave out limitations in this doc. It can be put somewhere else, but not in the first docs the user sees.

Reply:
Why would we want to hide that info further away? As a new user, I would appreciate learning about potential limitations as early as possible to be able to determine of the tool is suitable for my needs.

I’m not suggesting to put it very prominently at the beginning. If needed, it might be a link to another page as well.